containers/Makefile

```diff
@@ -110,20 +110,20 @@ build/concent-api/requirements.lock: build/concent-api/concent-api.tar
 build/concent-api/testing.pref: ../concent-builder/files/testing.pref
 	cp "$<" "$@"
 
-build/concent-api/concent-api.tar: build/repositories/concent/  build/virtualenv/
+build/concent-api/concent-api.tar: build/repositories/concent  build/virtualenv/
```

Comment:
Cosmetic changes like this should be in a separate commit, especially when there's more of them like in this case.

Reply:
Done.